simulariumio/tests/conftest.py

```diff
@@ -131,9 +131,11 @@ def three_default_agents() -> TrajectoryData:
             display_data={
                 "C": DisplayData(
                     name="C",
+                    display_type=DISPLAY_TYPE.SPHERE,
```

Comment:
Is this change because you removed the default setting?

Reply:
yes